.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -75,18 +75,6 @@ If the code communicates with devices, web services, or third-party tools:
 - [ ] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.
 - [ ] Untested files have been added to `.coveragerc`.
 
-The integration reached or maintains the following [Integration Quality Scale][quality-scale]:
-<!--
```

Comment:
Should we leave this commented out part and include the link?

Reply:
Nah its ok, let's remove it for now.